recipes/libraw/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+from conans import ConanFile, tools, CMake
+import os
+
+class LibRawConan(ConanFile):
+    name = "libraw"
+    description = "LibRaw is a library for reading RAW files obtained from digital photo cameras (CRW/CR2, NEF, RAF, DNG, and others)."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.libraw.org/"
+    license = "CDDL-1.0/LGPL-2.1-only"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    requires = [
+        "libjpeg/9d",
+        "lcms/2.10"
+    ]
+
+    version = "0.19.5" # TODO remove
+
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("LibRaw-" + self.version, self._source_subfolder)
+
+    def configure_cmake(self):
+        cm = CMake(self)
+        
+        cm.configure()
+        return cm
```

Comment:
```suggestion
    def _configure_cmake(self):
        if self._cmake:
            return self._cmake
        self._cmake = CMake(self)
        self._cmake.configure()
        return self._cmake
```

caching the cmake object avoid a second configure making the builds faster